src/selectors/layers.js

```diff
@@ -44,13 +44,14 @@ export const getLayers = createSelector(
       ];
       const start = (prevBound[1] + currentBound[0]) / 2;
       const end = (currentBound[1] + nextBound[0]) / 2;
+      const rectWidth = Math.max(width, height) * 5;
 
       return {
         id,
         name: layerName[id],
-        x: -width / 2,
+        x: (rectWidth - width) / -2,
```

Comment:
Not an issue but I think `(width - rectWidth) / 2` might work here too and avoid that negative denominator.

Reply:
Ah interesting, good shout thanks! Probably won't bother now but good to bear in mind.